articles/virtual-machines/monitor-vm-reference.md

```diff
@@ -31,20 +31,22 @@ For an example that shows how to collect the *Percentage CPU* metric from a VM,
 ### VM availability metric (preview)
 The VM availability metric is currently in public preview. This metric value indicates whether a machine is currently running and available. You can use the metric to trend availability over time and to alert if the machine is stopped. VM availability displays the following values.
 
+To learn how to use the VM availability metric to monitor Azure Virtual Machine availability, see [Use Azure Monitor to monitor Azure Virtual Machine availability](flash-azure-monitor.md).
+
 | Value | Description |
 |:---|:---|
 | 1 | VM is running and available. |
 | 0 | VM is unavailable. The VM could be stopped or rebooting. If you shut down a VM from within the VM, it emits this value. |
 | Null (dashed line) | State of the VM is unknown. If you stop a VM from the Azure portal, CLI, or PowerShell, it immediately stops emitting the availability metric, and you see null values. |
 
+**Context** dimension: Informs whether VM availability was influenced due to Azure or user orchestrated activity. It can assume values of _Platform | Customer | Unknown_.
```

Comment:
```suggestion
**Context dimension** informs whether VM availability was influenced due to Azure or user orchestrated activity. It can assume values of *Platform*, *Customer*, or *Unknown*.
```